spec/instant.html

```diff
@@ -647,5 +623,40 @@ <h1>TemporalInstantToString ( _instant_, _timeZone_, _precision_ )</h1>
         1. Return the string-concatenation of _dateTimeString_ and _timeZoneString_.
       </emu-alg>
     </emu-clause>
+    <emu-clause id="sec-temporal-differencetemporalinstant" type="abstract operation">
+      <h1>
+        DifferenceTemporalInstant (
+          _operation_: ~since~ or ~until~,
+          _instant_: a Temporal.Instant,
+          _other_: an ECMAScript language value,
+          _options_: an ECMAScript language value,
+        ): either a normal completion containing a Temporal.Duration or an abrupt completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It computes the difference between the two times represented by _instant_ and _other_, optionally rounds it, and returns it as a Temporal.Duration object.</dd>
+      </dl>
+      <emu-alg>
+        1. Set _other_ to ? ToTemporalInstant(_other_).
+        1. If _operation_ is ~until~, then,
+          1. Let _first_ be _instant_.
+          1. Let _second_ be _other_.
+        1. Else,
+          1. Let _first_ be _other_.
+          1. Let _second_ be _instant_.
+        1. Set _options_ to ? GetOptionsObject(_options_).
+        1. Let _smallestUnit_ be ? ToSmallestTemporalUnit(_options_, « *"year"*, *"month"*, *"week"*, *"day"* », *"nanosecond"*).
+        1. Let _defaultLargestUnit_ be ! LargerOfTwoTemporalUnits(*"second"*, _smallestUnit_).
+        1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, « *"year"*, *"month"*, *"week"*, *"day"* », *"auto"*, _defaultLargestUnit_).
+        1. Perform ? ValidateTemporalUnitRange(_largestUnit_, _smallestUnit_).
+        1. Let _roundingMode_ be ? ToTemporalRoundingMode(_options_, *"trunc"*).
+        1. Let _maximum_ be ! MaximumTemporalDurationRoundingIncrement(_smallestUnit_).
```

Comment:
Other similar AOs do this: 
```
        1. If _operation_ is ~since~, then
          1. Set _roundingMode_ to ! NegateTemporalRoundingMode(_roundingMode_).
```
Why isn't this step required here too?

Reply:
because it is currently not in 
https://tc39.es/proposal-temporal/#sec-temporal.instant.prototype.since

(why is it not there? I have no clue. I just did a garbage-in-garbage-out refactor. If the current spec is wrong, then my refactor produce the same wrongful result. I am not here to correct mistake, just to merge redundancy in spec text)

Reply:
@ptomato and I discussed this, and the answer was that the arg swap makes it unnecessary to negate the rounding mode, but if the "multiply by sign" approach were used instead (as it is in other types' AOs) then negating the rounding mode would be required.

So this is not a bug above.